ci/run-docker.sh

```diff
@@ -18,7 +18,7 @@ run() {
       --volume `pwd`/target:/checkout/target \
       --workdir /checkout \
       --privileged \
-      --env RUSTFLAGS \
+      --env RUSTFLAGS="-Clink-arg=-L/usr/lib/x86_64-linux-musl/" \
```

Comment:
Does using `RUSTFLAGS="-Lnative=..."` not work?

Reply:
I agree this is better if it works. I amended the commit :crossed_fingers: